debian/control

```diff
@@ -0,0 +1,431 @@
+ language in the world. The main goals of MariaDB are speed, robustness and
+ ease of use.
+ .
+ This package includes the client binaries and the additional tools
+ innotop and mysqlreport.
+
+Package: mariadb-server-core-10.1
+Architecture: any
+Depends: libmariadbclient18 (>= ${binary:Version}),
+         mariadb-common (>= ${source:Version}),
+         ${misc:Depends},
+         ${shlibs:Depends}
+Provides: mysql-server-core,
+          mysql-server-core-5.1,
+          mysql-server-core-5.5,
+          mysql-server-core-5.6,
+          virtual-mysql-server-core
+Conflicts: mariadb-server-core-10.0,
+           mariadb-server-core-5.1,
+           mariadb-server-core-5.2,
+           mariadb-server-core-5.3,
+           mariadb-server-core-5.5,
+           mysql-server-5.0,
+           mysql-server-core-5.0,
+           mysql-server-core-5.1,
+           mysql-server-core-5.5,
+           mysql-server-core-5.6,
+           virtual-mysql-server-core
+Replaces: mariadb-server-core-10.0,
+          mariadb-server-core-5.1,
+          mariadb-server-core-5.2,
+          mariadb-server-core-5.3,
+          mariadb-server-core-5.5,
+          mysql-server-5.0,
+          mysql-server-core-5.0,
+          mysql-server-core-5.1,
+          mysql-server-core-5.5,
+          mysql-server-core-5.6,
+          virtual-mysql-server-core
+Description: MariaDB database core server files
+ MariaDB is a fast, stable and true multi-user, multi-threaded SQL database
+ server. SQL (Structured Query Language) is the most popular database query
+ language in the world. The main goals of MariaDB are speed, robustness and
+ ease of use.
+ .
+ This package includes the core server files, as used by Akonadi.
+
+Package: mariadb-test-10.1
+Architecture: any
+Depends: mariadb-client-10.1 (= ${binary:Version}),
+         mariadb-server-10.1 (= ${binary:Version})
+Suggests: patch
+Conflicts: mariadb-galera-server-5.5 (<< 5.5.33),
+           mariadb-server-5.5 (<< 5.5.33),
+           mariadb-test (<< ${source:Version}),
+           mariadb-test-10.0,
+           mariadb-test-5.1,
+           mariadb-test-5.2,
+           mariadb-test-5.3,
+           virtual-mysql-testsuite
+Replaces: mariadb-test (<< ${source:Version}),
+          mariadb-test-10.0,
+          mariadb-test-5.1,
+          mariadb-test-5.2,
+          mariadb-test-5.3,
+          virtual-mysql-testsuite
+Provides: virtual-mysql-testsuite
+Description: MariaDB database regression test suite
+ MariaDB is a fast, stable and true multi-user, multi-threaded SQL database
+ server. SQL (Structured Query Language) is the most popular database query
+ language in the world. The main goals of MariaDB are speed, robustness and
+ ease of use.
+ .
+ This package includes the regression test suite.
+
+Package: mariadb-server-10.1
+Architecture: any
+Suggests: mailx, mariadb-test, netcat-openbsd, socat, tinyca
+Recommends: libhtml-template-perl
```

Comment:
I mean "libhtml-template-perl", "that" referred to the line directly above the comment.
But you'll probably reply that it's in upstream and you want to leave it for the later step. I'll answer to than at once — ok, let's leave it for later :)